### ISSUE TITLE
The std::old_io in the examples

### ISSUE THREAD

**Issue by prewk** [PistonDevelopers/image#436]
Hi, I'm completely new at Rust so yell at me if I'm doing something wrong, but I can't open an image due to the use of `std::old_io`.

````rust
extern crate image;

use std::old_io::File;
use image::GenericImage;

fn main() {
    let img = image::open(&Path::new("images/medium.jpg")).unwrap();

    println!("{:?}", img.dimensions());
}
````

"src/main.rs:3:5: 3:22 error: unresolved import `std::old_io::File`. Could not find `old_io` in `std`"

**Comment by Furyhunter** [comment]
Have you tried switching `use std::old_io::File` to `use std::fs::File`? Also, it doesn't appear as though it's used in this example anyway, so you could just remove it as well.

**Comment by nwin** [comment]
Thanks for bringing this up, the examples really need to be updated…

**Comment by prewk** [comment]
Yeah, I ended up using `let img = image::open("images/medium.jpg").unwrap();` which worked (sort of, the JPEG support wasn't what I'd hoped for but that's for another day I guess.)

**Comment by nwin** [comment]
Closed via #437.
